src/coreclr/jit/llvm.h

```diff
@@ -64,6 +64,27 @@ struct SsaPairHash
     }
 };
 
+struct TypeDescriptor
+{
+    size_t                fieldCount;
+    CORINFO_FIELD_HANDLE* fields;
+};
+
+struct StructDesc;
+
+struct FieldDesc
+{
+    int                  fieldOffset;
+    CORINFO_CLASS_HANDLE classHandle;
+    CorInfoType          corType;
+};
+
+struct StructDesc
+{
+    size_t     fieldCount;
+    FieldDesc* fieldDesc;
+};
```

Comment:
It would be nice to make `StructDesc` and `FieldDesc` into "proper" types, with constructors, private members, accessor methods, etc.

Reply:
ok, in the same files (llvm.cpp/llvm.h) or add new ones?